css/templates/gallery/components/_button.css

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright 2017 The AMP Start Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.ampstart-btn {
+  display: inline-flex;
+  height: 3.6rem;
+  padding: calc((30 / 16) * 1rem);
+  font-size: 1.1rem;
+  letter-spacing: 0.02em;
+  font-weight: 600;
+  border: 1px solid var(--color-primary);
+  color: var(--color-white);
+  transition: transform 0.1s, box-shadow 0.1s;
+  box-shadow: 0 0.3rem 1rem rgba(6, 7, 22, 0.1);
+}
+
+button.ampstart-btn {
```

Comment:
do  you need this specificity, also are you sure you want to set it to 0?

Reply:
Yes, since there's a conditional in our template when we either set that class on an `<a>` element, or a `<button>` element. The `button` element needs the line-height removed to be vertically centred, but applying that to the `a` element breaks it, so I've scoped the class.